pom.xml

```diff
@@ -361,6 +365,13 @@
 					</execution>
 				</executions>
 			</plugin>
+
+			<plugin>
```

Comment:
Any specific reason why flyway wasn't used as a dependency, but instead as a build plugin?

By using the plugin, you rely on a maven stage that needs to be executed prior to the application itself. After the application is built, there's no more maven, so it ends up being useless.

Please check the [following pom example](https://github.com/paulushcgcj/simple-webflux/blob/main/pom.xml#L184) and the [following yml/properties example](https://github.com/paulushcgcj/simple-webflux/blob/main/src/main/resources/application.yml#L19) where the actual dependency is being used.

Keep in mind that the above examples use some extra configurations that aren't required when using JDBC, as the example project is using r2dbc instead. By default, flyway will use the same configuration being used by spring JPA.

Reply:
I didn't realize this wouldn't work in our deploys. I also didn't know Spring accepted Flyway configuration.

This is better, indeed.